core/src/test/kotlin/org/evomaster/core/search/gene/AbstractGeneTest.kt

```diff
@@ -0,0 +1,18 @@
+package org.evomaster.core.search.gene
+
+import org.evomaster.core.search.service.Randomness
+
+abstract class AbstractGeneTest {
+
+    private val genes = GeneSamplerForTests.geneClasses
```

Comment:
shouldn't this be protected? also, isn't the name confusing? I mean, we are dealing with the classes of genes, not gene instance.
Maybe refactor into: `protected val geneClasses = GeneSamplerForTests.geneClasses`